### ISSUE TITLE
Invalid totp passcode

### ISSUE THREAD

**Issue by 0xCCD** [go-vikunja/app#6]
Hi I use a self hosted version of vikunja and try to get the app working,
unfortunately everytime I try to Login via app 

I get this error on app:

```
login failed! Please chack your server url and credentials. Invalid totp passcode
```

and this on server:

```
▶ v1/Login 0a0 Invalid totp passcode
▶ 80.187.100.57  POST 412 /api/v1/login 126.167109ms - Dart/2.18 (dart:io)
```

Could it be related to the 2FA ? 

Is 2FA currently not supported by the app ?

**Comment by Benimautner** [comment]
2FA is currently not supported. You could try to use the 'login via frontend' button and check if that works but I just recently found out that when the token expires the frontend login fails completely, so you will probably have to wait until the frontend login is fixed.

**Comment by 0xCCD** [comment]
Login via frontend works but it does not remember wme when i come back to the app, so it's only partially working for me, I am waiting for 2FA support then.
Thank you

**Comment by Benimautner** [comment]
2FA is implemented now (in v0.0.13-alpha). You can either try to use it natively by entering your API url, user and password and then waiting for a popup that will query the code...
... or you can use the webview login as that should be fixed as well now.

Please let me know if your issues are fixed!

**Comment by 0xCCD** [comment]
Works fine, thank you !!!!
